langkit/envs.py

```diff
@@ -166,6 +172,31 @@ def add_to_env_kv(key: AbstractExpression,
     )
 
 
+def add_to_env_by_name(key: AbstractExpression,
+                       val: AbstractExpression,
+                       name_expr: AbstractExpression,
```

Comment:
Could we remove the `_expr` part from these two parameter names? It feels weird for the users when calling this function using keyword argument notation.

Reply:
We already have such suffixes in other functions from `langkit.envs`. I’m ok with changing them too, but that’s going to be in another commit.